data.py

```diff
@@ -1,3 +1,4 @@
+#! usr/bin/python
```

Comment:
I like the idea, but let's go with `#! usr/bin/env python` just in case they don't have it installed in their user area. And then could you apply it to the other files?